src/BlockInfo.h

```diff
@@ -1,6 +1,7 @@
 
 #pragma once
 
+#include "ChunkDef.h"
```

Comment:
Block info is included in globals, so this PR won't change anything.

Reply:
I may not be looking properly but I couldn't see how it was included, could you explain?

Reply:
Oh sorry, I was looking at one of the other `Globals.h` files. Never mind then.